<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConcatenateOptionsDialog</class>
 <widget class="QDialog" name="ConcatenateOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Concatenate options</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Skip n first lines:</string>
       </property>
       <property name="buddy">
        <cstring>skipNFirstLines_lineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="skipNFirstLines_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="deleteOriginalFiles_checkBox">
     <property name="text">
      <string>&amp;Delete original files, be careful !</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_pushButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_pushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>skipNFirstLines_lineEdit</tabstop>
  <tabstop>deleteOriginalFiles_checkBox</tabstop>
  <tabstop>OK_pushButton</tabstop>
  <tabstop>Cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
